### PR TITLE
feat: Add SQL Server support

### PR DESCRIPTION
**Pull request by matsonj**

### CODE REVIEW

**Comment by matsonj** [comment]
the meltano.yml has a bunch of changes bc the meltano parser changed the spacing. sorry about that.

File: meltano.yml [hunk 1]
```diff
@@ -2,241 +2,287 @@ version: 1
...
+    - name: settings.mssql.user
+      env: MSSQL_USER
+      kind: string
+      label: MSSQL user name
+    - name: settings.mssql.password
+      env: MSSQL_PASSWORD
+      kind: password
+      label: MSSQL Password
+    - name: settings.mssql.trust_server_certificate
+      env: MSSQL_CERTIFICATE
```
Comment (edgarrmondragon) [inline meltano.yml:257]: Evidence seems to expect these settings to be nested inside an `options` object:

```suggestion
    - name: settings.mssql.options.trust_server_certificate
      env: MSSQL_TRUST_SERVER_CERTIFICATE
```

https://github.com/evidence-dev/evidence/blob/main/packages/mssql/index.cjs#L101-L105

File: meltano.yml [hunk 2]
```diff
@@ -2,241 +2,287 @@ version: 1
...
+      label: Trust server certificate
+      description: |
+        SQL Server server certificate
+    - name: settings.mssql.port
+      env: MSSQL_PORT
+      kind: string
+      value: false
+      label: Port
+    - name: settings.mssql.encrypt
+      env: MSSQL_ENCRYPT
```
Comment (edgarrmondragon) [inline meltano.yml:268]:
```suggestion
    - name: settings.mssql.options.encrypt
      env: MSSQL_ENCRYPT
```

**Review by edgarrmondragon — CHANGES_REQUESTED** [review]
Thanks @matsonj! I've left some comments based on the implementation I see on the Evidence codebase.

**Comment by matsonj** [comment]
Thanks @edgarrmondragon! I missed those somehow.

**Review by edgarrmondragon — APPROVED** [review]
Thanks @matsonj! We'll have to port it to the Hub (cc @pnadolny13)

**Pull request merged**